server/src/servershot.js

```diff
@@ -165,7 +165,7 @@ class Shot extends AbstractShot {
     this.ownerId = ownerId;
   }
 
-  isOwner(deviceId, accountId) {
+  checkOwner(deviceId, accountId) {
```

Comment:
This is redundant and can be removed

Reply:
I agree, we shouldn't have different kinds of checks or checking functions around, we should have One Right Way